src/commands/command.js

```diff
@@ -27,10 +27,34 @@ export default class DiscordCommand {
       this.discordClient = discordClient;
   }
 
-  get hasMentions() {
+  get member() {
+    return this.message.member;
+  }
+
+  get hasChannelMentions() {
+    return this.message.mentions.channels.size > 0;
```

Comment:
to cut down on redundancy and possible API breakage if discord ever changes their `Message` Class structure, I'd just use `this.channelMentions.size > 0` here so you can maintain the API bridging in that one getter

```suggestion
    return this.channelMentions.size > 0;
```